miner/algo_common.go

```diff
@@ -519,7 +556,9 @@ func (envDiff *environmentDiff) commitSBundle(b *types.SimSBundle, chData chainD
 	return nil
 }
 
-func (envDiff *environmentDiff) commitSBundleInner(b *types.SBundle, chData chainData, interrupt *int32, key *ecdsa.PrivateKey) error {
+func (envDiff *environmentDiff) commitSBundleInner(
```

Comment:
Again, convention of the codebase is otherwise and without both a discussion and a good reason we shouldn't change it